# The state of a unicycle model robot (x, y, theta)
float64 x
float64 y
float64 theta